lib/semigrp.gi

```diff
@@ -422,20 +422,29 @@ InstallMethod(DisplaySemigroup, "for finite semigroups",
     [IsTransformationSemigroup],
 function(S)
 
-    local dc, i, len, sh, D, layer, displayDClass;
+    local dc, i, len, sh, D, layer, displayDClass, n;
 
     displayDClass:= function(D)
-        local h, sh;
+        local h, nrL, nrR;
         h:= GreensHClassOfElement(AssociatedSemigroup(D),Representative(D));
         if IsRegularDClass(D) then
             Print("*");
+        else
+            Print(" ");
+        fi;
+        nrL := Size(GreensRClassOfElement(AssociatedSemigroup(D),
+                                          Representative(h))) / Size(h);
+        nrR := Size(GreensLClassOfElement(AssociatedSemigroup(D),
+                                          Representative(h))) / Size(h);
+        Print("[H size = ", Size(h), ", ", nrL, " L-class");
+        if nrL > 1 then
+          Print("es");
```

Comment:
You use 2 spaces for indentation here, while earlier in this function, 4 spaces are used...

Reply:
Good point @fingolfin, I'll unify it to 4 spaces (the original number used by this method).